### PR TITLE
Add rollup tables for places and sources

### PR DESCRIPTION
**Pull request by c-w**
[project-fortis#50](https://github.com/CatalystCode/project-fortis/issues/50)

### CODE REVIEW

File: ops/storage-ddls/cassandra-setup.cql [hunk 1]
```diff
@@ -140,6 +140,27 @@ CREATE INDEX ON computedtopics (period);
...
 
+CREATE TABLE topplaces (
+    placeid text,
+    placename text,
+    periodstartdate timestamp,
+    periodenddate timestamp,
+    periodtype text,
+    period text,
+    mentions int,
+    PRIMARY KEY ((placeid), periodstartdate, periodenddate)
```
Comment (jcjimenez) [inline ops/storage-ddls/cassandra-setup.cql]: Pardon the remedial question, but does this extra set of parenthesis around `placeid` mean something in Cassandra?

**Review by jcjimenez — APPROVED** [review]
LGTM

File: ops/storage-ddls/cassandra-setup.cql (hunk 1 above)
Comment (c-w) [reply ops/storage-ddls/cassandra-setup.cql]: The columns in the parens are the "table key" and the other ones are the "partition key". [More information](https://stackoverflow.com/a/24953331/3817588).

**Comment by c-w** [comment]
Maybe there's a better way to do this.

**Pull request closed without merging**